docker/Dockerfile.DeviceManager

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+
+RUN mkdir -p /etc/deviceManager/configs /etc/deviceManager/certs /etc/deviceManager/registryStore /var/log/deviceManager
+RUN touch /var/log/deviceManager/svc-account-session.log /var/log/deviceManager/svc-aggregation.log \
+    /var/log/deviceManager/svc-api.log /var/log/deviceManager/svc-events.log /var/log/deviceManager/svc-systems.log \
+    /var/log/deviceManager/svc-task.log /var/log/deviceManager/svc-device-manager.log
+COPY apps/* /usr/local/bin/
+COPY build/runODIMServices.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/runODIMServices.sh /usr/local/bin/svc*
+RUN chmod +w /var/log/deviceManager/svc*
+COPY lib-utilities/config/*.json /etc/deviceManager/configs/
+COPY lib-utilities/etc/* /etc/deviceManager/registryStore/
+COPY lib-messagebus/platforms/platformconfig.toml /etc/deviceManager/configs/kafkaConfig.toml
+COPY svc-device-manager/config/config.yml /etc/deviceManager/configs/
+COPY build/certs/* /etc/deviceManager/certs/
+
```

Comment:
Are certs located in build/certs folder on the machine? Not in /etc/deviceManager/certs?

Reply:
Yes, certs should be placed inside build/certs folder of the root project due to Docker context location